BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/physicsOfResultsPUEO.py

```diff
@@ -55,85 +57,147 @@
 PassingWeights = defaultdict(list)
 TotalEvents = defaultdict(list)
 RawWeights = defaultdict(list)
-success_runs = []
 
 var = ['trigg', 'passed', 'passWeight', 'rawWeights', 
        'position', 'neutrinoFlavor', 
        'interactionLength', 'interactionCrossSection',
        'interactionStrength', 'showerPnuEv', 'maxEField', 
-       'maxEFieldFreq', 'nu_e', 'nu_m', 'nu_t']
+       'maxEFieldFreq', 'nu_e', 'nu_m', 'nu_t', 'RFx', 'RFy', 
+       'RFz', 'RFdirCosTheta', 'RFdirTheta', 'nuDirCosTheta', 
+       'nuDirTheta']
+
+
+#Create a multigraph for the signalAtDetector TGraphs
+#signalAtDetectorMother = ROOT.TMultiGraph()
+
 
 for x in var:
     var_dict['{0}'.format(x)] = []
 
 #Functions
 def getFiles(source, energy, indiv):
-    
+    counter = 1
     # The root files will be located in the 
     # $WorkingDir/Run_Outputs/$RunName/Root_Files/$genNum_Root_Files/ directory
     # The root files will be named IceFinal_$genNum_$indiv_$seed.root
 
     root = source
     print(root)
-    pattern = "IceFinal_*_{}_*.root".format(indiv)
-
+    all_tree_pattern = "IceFinal_allTree_*_{}_*.root".format(indiv)
+    pass_tree_pattern = "IceFinal_passTree_*_{}_*_1.root".format(indiv)
 
     # Walk through the directory and find the root files
     for path, subdirs, files in os.walk(root):
         j = 0
         for name in files:
-            if fnmatch(name, pattern):
+            if fnmatch(name, all_tree_pattern):
                 print(os.path.join(path, name))
                 
                 try:
                     # Open the root file and assign the trees to variables
                     fileName = os.path.join(path, name)
                     IceFinalFile = ROOT.TFile(fileName)
                     
-                    nuWeights = []
-                    nuPasses = []
 
                     allTree = IceFinalFile.allTree
                     allEvents = allTree.GetEntries()
-
+                    print(allEvents)
                     TotalEvents[energy].append(allEvents)
+                    
+                except:
+                    print("Error opening file")
```

Comment:
```suggestion
                except Exception as e:
                    print("Error opening file:", e)
```
Never use bare `except:` -- you should at least catch `Exception`, but catching more specific errors is better.